include/zephyr/logging/log_backend.h

```diff
@@ -341,6 +377,39 @@ static inline int log_backend_format_set(const struct log_backend *backend, uint
 }
 #endif
 
+/**
+ * @brief Notify a backend of an event.
+ *
+ * @param backend Pointer to the backend instance.
+ * @param event Event to be notified.
+ * @param arg Pointer to the argument(s).
+ */
+static inline void log_backend_notify(const struct log_backend *const backend,
+				      enum log_backend_evt event,
+				      union log_backend_evt_arg *arg)
+{
+#ifdef CONFIG_LOG_BACKEND_EVENT_NOTIFICATION_SUPPORT
+	__ASSERT_NO_MSG(backend != NULL);
+
+	if (backend->api->notify) {
```

Comment:
if (IS_ENABLED(CONFIG_LOG_BACKEND_EVENT_NOTIFICATION_SUPPORT) && backend->api->notify) would make this look cleaner. (i am aware that it may be my personal preference).

Reply:
Removed `#ifdef`.